cmd/zqd/listen/command.go

```diff
@@ -37,8 +38,12 @@ func New(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
 }
 
 func (c *Command) Run(args []string) error {
+	cwd, err := os.Getwd()
```

Comment:
We'll want a cli flag to set this, in a later PR.

Reply:
Yep, was trying to keep to the scope creep down on a pr where it admittedly is a little out of hand 😄 .

Reply:
I'll add a comment though.